tests/unit/challenges/test_utils.py

```
@@ -95,6 +95,29 @@ def test_sqs_queue_name_generator_empty_title(self):
         self.assertLessEqual(len(sqs_queue_name), 80)
 
+    def test_sqs_queue_name_fifo_suffix(self):
+        title = "Test Challenge"
+        challenge_pk = 1
+        sqs_queue_name = get_queue_name(title, challenge_pk, use_fifo=True)
+        self.assertTrue(sqs_queue_name.endswith(".fifo"))
+        self.assertLessEqual(len(sqs_queue_name), 80)
+
+    def test_sqs_queue_name_fifo_long_title(self):
+        title = "".join(
+            [random.choice(self.sqs_valid_characters) for i in range(256)]
```

Ruff (0.16.0)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)